python/observation-publisher/Makefile

```diff
@@ -4,6 +4,12 @@ ENVIRONMENT_CONFIG = "example-override"
 setup:
 	@echo "Setting up environment..."
 	@pip install -r requirements.txt --use-pep517
```

Comment:
do we still need this?

Reply:
`--use-pep517` is used mainly to suppress the warning.